requirements.txt

```diff
@@ -1 +1 @@
-honeybee-energy
\ No newline at end of file
+honeybee-energy
```

Comment:
I would add a minimum version here. Specially that this will only work fine with the most recent versions of honeybee-core that has the new approach for loading extensions.

Reply:
Ok. Will do. I was afraid to do this at one point because saying the version should be `==` to a number was creating a lot of headaches downstream in the dependency tree. Using `>=` seems t work well, though.